README.md

```diff
@@ -1,6 +1,11 @@
 three.js
 ========
 
+[![Latest NPM release][npm-badge]][npm-badge-url]
+[![License][license-badge]][license-badge-url]
+[![Dependencies][dependencies-badge]][dependencies-badge-url]
+[![Dev Dependencies][devDependencies-badge]][devDependencies-badge-url]
```

Comment:
Ah, I see. `devDependencies` simply conveys information about the "freshness" of our node `devDependencies` listed in `package.json`. The badge provided corresponds to [the data being recorded here](https://david-dm.org/mrdoob/three.js?type=dev) about each package's latest version.